packages/legacy/core/App/contexts/activity.tsx

```diff
@@ -144,13 +131,26 @@ export const ActivityProvider: React.FC<PropsWithChildren> = ({ children }) => {
       clearInactivityTimeoutIfExists()
       eventSubscription.remove()
     }
-  }, [clearInactivityTimeoutIfExists, lockOutUser, resetInactivityTimeout, timeoutInMilliseconds, agent, logger])
```

Comment:
`timeoutInMilliseconds` is a ref so it doesn't need to be added to the dependency array